src/hello/types.md

```diff
@@ -6,16 +6,20 @@ Even though variables do not need to have their types defined for primitive data
 
 Primitive types are types that do not need variables to be typecasted anyhow. This includes:
 
-* Numeric values: `1, 24.3, 3.14`
-* Strings: `"a", "abc"`
-* Booleans: `TRUE, FALSE` (which aren't anything different than the numerical values 1 and 0)
+* Numeric values: `1, 24.3, 3.14` (WARNING: DM doesn't use integers, it uses floats. Watch out for floating point precision errors.)
+* [Strings](../vars/strings.md): `"a", "abc"` (These can use single or double quotes, although usually a codebase will have a general form which you should follow.)
```

Comment:
Um, no, they can't.

Reply:
huh. really? I thought the path thing in single quotes was just from the add in. Who knew